api/base/settings.py

```diff
@@ -31,6 +31,10 @@
 # See https://docs.djangoproject.com/en/1.5/ref/settings/#allowed-hosts
```

Comment:
Maybe update the link so it goes to [the current docs](https://docs.djangoproject.com/en/1.10/ref/settings/#allowed-hosts) rather than 404 on the old ones?